todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/tasks/TasksViewModel.java

```diff
@@ -47,17 +43,17 @@
  * property changes. This is done by assigning a {@link Bindable} annotation to the property's
  * getter method.
  */
-public class TasksViewModel extends AndroidViewModel {
+public class TasksViewModel extends ViewModel {
 
     private final MutableLiveData<List<Task>> mItems = new MutableLiveData<>();
 
     private final MutableLiveData<Boolean> mDataLoading = new MutableLiveData<>();
 
-    private final MutableLiveData<String> mCurrentFilteringLabel = new MutableLiveData<>();
+    private final MutableLiveData<Integer> mCurrentFilteringLabel = new MutableLiveData<>();
 
-    private final MutableLiveData<String> mNoTasksLabel = new MutableLiveData<>();
+    private final MutableLiveData<Integer> mNoTasksLabel = new MutableLiveData<>();
 
-    private final MutableLiveData<Drawable> mNoTaskIconRes = new MutableLiveData<>();
+    private final MutableLiveData<Integer> mNoTaskIconRes = new MutableLiveData<>();
```

Comment:
@JoseAlcerreca Is it really needed to expose these objects as _Mutable_ LiveData?